tracer/src/Datadog.InstrumentedAssemblyGenerator/InstrumentedMethod.cs

```diff
@@ -101,6 +101,10 @@ internal static InstrumentedMethod ReadFromFile(string filePath)
                 filePath.Replace(TextFilePrefix, BinaryFilePrefix));
         }
 
+        /// <summary>
```

Comment:
This class can be internal, you can remove this

Reply:
Fixed in d6e34e581c70cc2271806c12db2c3d4f2caf78f1